### ISSUE TITLE
Support AppLens for scm site scenarios

### ISSUE THREAD

**Issue by ShekharGupta1988** [ShekharGupta1988/AppLensV2#457]
(Reported by Suwat)

One Scenario:
https://applensv2.azurewebsites.net/sites/IntuneStorageSnapshotSelfHost-SHAMSUA01/appanalysis/detectors/workerprocessrecycle

This site's scm process encountered worker process stop events due to file server failover. However, since we only look at main site process, we didnt capture this.

**Comment by ShekharGupta1988** [comment]
We dont need to beep in case of scm site but just adding data to view would be good to have.

**Comment by hforeste** [comment]
Have we tested if this works? I know if you pass hostname of scm site Observer will return correct data. Will some of our detectors work?

**Comment by ShekharGupta1988** [comment]
I dont think it will work as no where in our code, we are mapping ApplicationPool as ~1<SiteName> for scm site

**Comment by PraveenTB** [comment]
this is important scenario for webjob specific detector's. Aspaan assiging this to you to look when webjob detectors are built
